eclipsePlugin/about.properties

```diff
@@ -1,4 +1,6 @@
 blurb=SpotBugs \n\
 \n\
-(c) SpotBugs Project. Licensed under the LGPL.\n\
-https://spotbugs.github.io\n\
+Copyright (c) 2017 SpotBugs Project. All rights reserved.\n\
+Licensed under the LGPL.\n\
+Copyright 2003-2008 by the University of Maryland. \n\
```

Comment:
I would move the two copyright lines together. Also, why is the date range (2003-2008) different from the ranges used below (2006-2011)?